server/tests/conftest.py

```diff
@@ -311,7 +311,7 @@ async def with_preloading(sdb, mdb, mdb_rw, pdb, rdb, with_preloading_enabled):
     if not with_preloading_enabled:
         return False
 
-    mc_preloader = MemoryCachePreloader()
+    mc_preloader = MemoryCachePreloader(60)
```

Comment:
Yeah, we definitely need unit tests of the refresh. Maybe in a separate PR, as you prefer.

Reply:
Indeed, I created a separate ticket: https://athenianco.atlassian.net/browse/DEV-2499